frontend/src/components/elements/PlotlyChart/CustomTheme.tsx

```diff
@@ -0,0 +1,471 @@
+      subtitleColor: colors.bodyText,
+      ...themeFonts,
+      font: {
+        family: genericFonts.bodyFont,
+        size: fontSizes.smPx + 2,
+        color: colors.headingColor,
+      },
+      // text: (layout.title.text) ? `<b>${layout.title.text}</b>` : undefined,
+      x: 0,
+    },
+    colorway: hasLightBackgroundColor(theme)
+      ? [
+          "#0068C9",
+          "#83C9FF",
+          "#FF2B2B",
+          "#FFABAB",
+          "#29B09D",
+          "#7DEFA1",
+          "#FF8700",
+          "#FFD16A",
+          "#6D3FC0",
+          "#D5DAE5",
+        ]
+      : [
+          "#83C9FF",
+          "#0068C9",
+          "#FFABAB",
+          "#FF2B2B",
+          "#7DEFA1",
+          "#29B09D",
+          "#FFD16A",
+          "#FF8700",
+          "#6D3FC0",
+          "#D5DAE5",
+        ],
+    legend: {
+      // side: "left",
+      // orientation: "h",
+      // yanchor: "bottom",
+      // xanchor: "left",
+      // y: -.25,
+      title: {
+        font: {
+          size: fontSizes.smPx,
+          color: getGreyText(theme),
+        },
+        side: "top",
+      },
+      valign: "top",
+      bordercolor: colors.transparent,
+      borderwidth: 0,
+      // tracegroupgap: 200,
+      font: {
+        color: getGreyText(theme),
+      },
+      labelColor: getGreyText(theme),
+      titleColor: getGreyText(theme),
+      ...themeFonts,
+    },
+    paper_bgcolor: colors.bgColor,
+    plot_bgcolor: colors.bgColor,
+    // yaxis: {
+    //   minor: {
+    //     showgrid: false,
+    //   }
+    // },
+    colorDiscreteSequence: hasLightBackgroundColor(theme)
+      ? [
+          "#0068C9",
+          "#83C9FF",
+          "#FF2B2B",
+          "#FFABAB",
+          "#29B09D",
+          "#7DEFA1",
+          "#FF8700",
+          "#FFD16A",
+          "#6D3FC0",
+          "#D5DAE5",
+        ]
+      : [
+          "#83C9FF",
+          "#0068C9",
+          "#FFABAB",
+          "#FF2B2B",
+          "#7DEFA1",
+          "#29B09D",
+          "#FFD16A",
+          "#FF8700",
+          "#6D3FC0",
+          "#D5DAE5",
+        ],
+    yaxis: {
+      // nticks: 10,
+      ticklabelposition: "outside",
+      tickmode: "auto",
+      // automargin: "left",
+      zeroline: false,
+      title: {
+        font: {
+          color: getGreyText(theme),
+        },
+      },
+      tickfont: {
+        color: getGreyText(theme),
+      },
+      automargin: true,
+    },
+    xaxis: {
+      standoff: 100,
+      // nticks: 8,
+      showgrid: false,
+      rangeselector: {
+        bordercolor: colors.transparent,
+        borderwidth: 0,
+      },
+      tickfont: {
+        color: getGreyText(theme),
+        size: fontSizes.smPx,
+      },
+      title: {
+        font: {
+          color: getGreyText(theme),
+          size: fontSizes.smPx,
+        },
+      },
+      zeroline: false,
+      automargin: true,
+    },
+    margin: {
+      pad: 5,
+      r: 10,
+      // t: 0,
+    },
+    hoverlabel: {
```

Comment:
You can probably slightly improve the tooltip by settings some additional properties:

```typescript
hoverlabel: {
      bgcolor: colors.bgColor,
      bordercolor: colors.fadedText10,
      font: {
        color: getGreyText(theme),
        family: genericFonts.bodyFont,
        size: fontSizes.sm,
      },
    },
```